### DISCUSSION TITLE
Privacy is not the only issue, let's address all of them

### DISCUSSION THREAD

**Discussion by JYamihud** [tenacityteam/tenacity-legacy#207] [General]
Surveillance as in Telemetry that MuseGroup try to push on us, not realizing that people will f**k them and go, is a very big issue. Since the Snowden revelations it's a very strong argument of the Free Software. But it's not the only malware. And I don't want my Tenacity, where I gonna edit my songs and vocal recording to have any malware at all.

Malware types ( with links ):

 - [Addictions](https://www.gnu.org/proprietary/proprietary-addictions.html)
 - [Back Doors](https://www.gnu.org/proprietary/proprietary-back-doors.html)
 - [Censorship](https://www.gnu.org/proprietary/proprietary-censorship.html)
 - [Coverups](https://www.gnu.org/proprietary/proprietary-coverups.html)
 - [Deception](https://www.gnu.org/proprietary/proprietary-deception.html)
 - [Digital Restrictions Management](https://www.gnu.org/proprietary/proprietary-drm.html)
 - [Fraud](https://www.gnu.org/proprietary/proprietary-fraud.html)
 - [Incompatibility](https://www.gnu.org/proprietary/proprietary-incompatibility.html)
 - [Insecurity](https://www.gnu.org/proprietary/proprietary-insecurity.html)
 - [Interference](https://www.gnu.org/proprietary/proprietary-interference.html)
 - [Manipulation](https://www.gnu.org/proprietary/proprietary-manipulation.html)
 - [Obsolescence](https://www.gnu.org/proprietary/proprietary-obsolescence.html)
 - [Sabotage](https://www.gnu.org/proprietary/proprietary-sabotage.html)
 - [Subscriptions](https://www.gnu.org/proprietary/proprietary-subscriptions.html)
 - [Surveillance](https://www.gnu.org/proprietary/proprietary-surveillance.html)
 - [Tethers](https://www.gnu.org/proprietary/proprietary-tethers.html)

I excluded [Jails](https://www.gnu.org/proprietary/proprietary-jails.html) and [Tyrants](https://www.gnu.org/proprietary/proprietary-tyrants.html) since it's about hardware and Operating Systems and not about software on those systems. 

I suggest that we make a "Privacy Policy" of our own. But naming it something like "User Respect Policy". And stating in details how we are going to avoid all of the malware listed above.

Thoughts?

**Comment by Songtech-0912** [comment]
@JYamihud I do like your idea of a "user respect policy". We could list those things out, sure. But a comprehensive list may not be that important, given that none of the things you mentioned are likely to (ever) get into Tenacity. Last time I checked, Tenacity has NONE of these things? Let's go through them:

- Addictions: In what way is Tenacity going to cause addictions? (Except for people like us who are obsessed with making Tenacity better)
- Back doors: There are no back doors into Tenacity. There is no way to remotely activate the recording functions of Tenacity without some significant modification or unless you run it in the background headlessly, which would only be possible if you yourself chose to do that.
- Censorship: There is no way to implement DRM-based stuff in Tenacity AFAIK (I could be wrong) and/or to destroy/disable access to specific audio content in Tenacity
- Coverups: So long as we keep project governance + decision process transparent and prevent corporate interests from taking root, that's not going to happen!
- We've removed most (if not all) data collection stuff already, there might be some left but we can root that out easily, no one's preventing us from doing that
- Deception: Our community is 100% a free-software-centric one, I wouldn't understand how deception would go into Tenacity if we aren't marketting it and we aren't lying about its capabilities
- DRM: None, for aforementioned reasons
- Fraud: We're not selling stuff here
- Incompatibility: Tenacity's source is always available, so this isn't going to be a problem anytime soon as users can compile their own versions to their own host OS's
- Insecurity: Possible, but I don't have any idea of how Tenacity could be insecure to the point of being damaging to the user
- Interference: No PR that infringes user privacy will ever go through
- Manipulation: We don't do that
- Obsolescence: No? Audacity can run on quite legacy hardware and you always have access to past versions of the code so that will never really be an issue?
- Sabotage: No, for pretty clear reasons - any attempt at sabotage will be in the code and will obviously be removed
- Subscriptions: No! We hate subscriptions! We are also no Adobe, so likelihood is negligible
- Surveillance: No
- Tethers: No as we publish all website code and any server code will be free software as well and self-hostable

Tell me if I've got any of these wrong, but I'm pretty sure they are all non-existent here.
